pkg/rpc/client.go

```diff
@@ -201,15 +179,18 @@ func (c *Client) GetBlockProduction(ctx context.Context, firstSlot *int64, lastS
 	// convert to BlockProduction format:
 	hosts := make(map[string]BlockProductionPerHost)
 	for id, arr := range resp.Result.Value.ByIdentity {
-		hosts[id] = BlockProductionPerHost{
-			LeaderSlots:    arr[0],
-			BlocksProduced: arr[1],
-		}
+		hosts[id] = BlockProductionPerHost{LeaderSlots: arr[0], BlocksProduced: arr[1]}
 	}
 	production := BlockProduction{
-		FirstSlot: resp.Result.Value.Range.FirstSlot,
-		LastSlot:  *resp.Result.Value.Range.LastSlot,
-		Hosts:     hosts,
+		FirstSlot: resp.Result.Value.Range.FirstSlot, LastSlot: *resp.Result.Value.Range.LastSlot, Hosts: hosts,
 	}
 	return &production, nil
 }
+
+func (c *Client) GetBalance(ctx context.Context, address string) (float64, error) {
+	var resp response[BalanceResult]
+	if err := c.getResponse(ctx, "getBalance", []interface{}{address}, &resp); err != nil {
+		return 0, err
+	}
+	return float64(resp.Result.Value / 1_000_000_000), nil
```

Comment:
nit: this [magic number ](https://en.wikipedia.org/wiki/Magic_number_(programming)) for the lamport to SOL converstion should be a const.

Reply:
fair nit, implemented soon